src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -11,6 +13,15 @@ public class AuthenticationService {
      * Return false in any other cases.
      */
     public boolean login(String email, String password) {
+
+        UserService us = new UserService();
+        User user = us.findByEmail(email);
+        if (user != null) {
+            if (password.equals(user.getPassword())) {
+                return true;
+            }
+        }
```

Comment:
this block of code could be easily simplified this way:
```suggestion
    return user != null && user.getPassword().equals(password);
```